vpr/src/place/place.cpp

```diff
@@ -326,14 +348,72 @@ static float analyze_setup_slack_cost(const PlacerSetupSlacks* setup_slacks);
 
 static e_move_result assess_swap(double delta_c, double t);
 
-static void get_non_updateable_bb(ClusterNetId net_id, t_bb* bb_coord_new);
-
-static void update_bb(ClusterNetId net_id, t_bb* bb_coord_new, t_bb* bb_edge_new, int xold, int yold, int xnew, int ynew);
+static void get_non_updateable_bb(ClusterNetId net_id,
```

Comment:
Comment every one of these.  Also comment why so many variations (give the big picture).

Reply:
In another PR...